src/bootstrap/middleware/nunjucks.js

```diff
@@ -33,6 +35,13 @@ const setup = (app, { views = "views", ...otherOptions } = {}) => {
     ...otherOptions,
   });
 
+  nunjucksEnv.addGlobal("addLanguageParam", frontendUi.addLanguageParam);
+  nunjucksEnv.addGlobal("contactUsUrl", frontendUi.contactUsUrl);
+  nunjucksEnv.addGlobal(
+    "May_2025_Rebrand",
+    process.env.May_2025_Rebrand === "true",
```

Comment:
Mind updating this env var to something like `MAY_2025_REBRAND_ENABLED` so that it's consistent with the others we have?